app/views/SelectedUsersView/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useRef } from 'react';
+import { FlatList, View, Text } from 'react-native';
+
+import { themes } from '../../lib/constants';
+import SearchBox from '../../containers/SearchBox';
+import I18n from '../../i18n';
+import { ISelectedUser } from '../../reducers/selectedUsers';
+import { useTheme } from '../../theme';
+import sharedStyles from '../Styles';
+import { useAppSelector } from '../../lib/hooks';
+import SelectedChipItem from './SelectedChipItem';
+
+const ITEM_WIDTH = 250;
+const getItemLayout = (_: any, index: number) => ({ length: ITEM_WIDTH, offset: ITEM_WIDTH * index, index });
```

Comment:
This is wrong.
The new Chip component has a dynamic size, so we can't calculate the positions.
Just remove the whole `getItemLayout` logic.